src/components/BackButton/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Icon } from 'antd';
+import './backbutton.css';
+
+const BackButton = () => {
+  return (
+    <div className="backButton">
```

Comment:
how can we implement the back button?
i mean where we should take action to it to back to the previous page

Reply:
Done